lib/ui/src/modules/ui/components/menu_item.stories.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
-import { action } from '@storybook/addon-actions';
+import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
what happened in all these files?

Reply:
I wanted to remove the cyclical dependencies warnings from lerna